internal/dev_server/dev_server.go

```diff
@@ -0,0 +1,9 @@
+package dev_server
+
+import (
+	"log"
+)
+
+func HelloWorld(accessTokenStr string) {
```

Comment:
I think a good next step here would be to port what's in dev_server/main.go into here.

Reply:
I'm also realizing it's probably better to pass around an api client instead of the api key. That way it'll work with different base urls.

Reply:
oh yeah we can/will, was just passing this along now to prove that we have it. the config file also already stores your base URL as well. 

here's an example of a request with the existing client we have: https://github.com/launchdarkly/ldcli/blob/f24a1ead0d5cd78c90d2b8e5bd754ca17678427a/cmd/flags/archive.go#L35-L49

Reply:
(but also I don't think we need a client for this cmd)